### PR TITLE
Keywords for management of components

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]
Looks good to me

**Review by sutaakar — DISMISSED** [review]

**Comment by jstourac** [comment]
LGTM, one proposal for an improvement would be to unify the code - so these two keywords would use one common implementation by addition also the `operation` parameter. WDYT @jiripetrlik ?

File: ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot [hunk 1]
```diff
@@ -329,3 +329,29 @@ Install Kserve Dependencies
...
+Set Component State
+    [Documentation]    Set component state in Data Science Cluster (state should be Managed or Removed)
+    [Arguments]    ${component}    ${state}
+    ${result} =    Run Process    oc get datascienceclusters.datasciencecluster.opendatahub.io -o name
+    ...    shell=true    stderr=STDOUT
+    IF    $result.stdout == ""
+        FAIL    Can not find datasciencecluster
+    END
+    Set Test Variable    ${cluster_name}    ${result.stdout}
+    ${result} =    Run Process    oc patch ${cluster_name} --type 'json' -p '[{"op" : "replace" ,"path" : "/spec/components/${component}/managementState" ,"value" : "${component}"}]'
```
Comment (jstourac) [inline ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot:342]:
```suggestion
    ${result} =    Run Process    oc patch ${cluster_name} --type 'json' -p '[{"op" : "replace" ,"path" : "/spec/components/${component}/managementState" ,"value" : "${state}"}]'
```

**Review by jstourac — CHANGES_REQUESTED** [review]
One comment, otherwise LGTM.

**Review by jstourac — APPROVED** [review]
LGTM, once tested properly (could you please share a CI?) I'm all for merge this.

**Comment by jiripetrlik** [comment]
I was able to successfully run this with Jenkins. @sutaakar @jstourac @jgarciao Can we merge now?

**Review by sutaakar — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**